pkgs/games/ironwail/default.nix

```diff
@@ -0,0 +1,96 @@
+{ lib
+, stdenv
+, SDL2
+, fetchurl
+, gzip
+, libvorbis
+, libmad
+, flac
+, libopus
+, opusfile
+, libogg
+, curl
+, libxmp
+, vulkan-headers
+, vulkan-loader
+, copyDesktopItems
+, makeDesktopItem
+, pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  pname = "ironwail";
+  version = "0.7.0";
+
+  src = fetchurl {
+    url = "https://github.com/andrei-drexler/ironwail/archive/refs/tags/v${version}.tar.gz";
+    sha256 = "sha256-NBG0wwQWqyGWQYJmiLKfxGxpDJLw7Kwf4EnYd33dOpU=";
+  };
+
+  sourceRoot = "${pname}-${version}/Quake";
+
+  nativeBuildInputs = [
+    copyDesktopItems
+    pkg-config
+    vulkan-headers
+  ];
+
+  buildInputs = [
+    gzip libvorbis libmad flac curl libopus opusfile libogg
+    libxmp vulkan-loader SDL2
```

Comment:
Are you sure about libraries being `buildInputs` ? Shouldn't they belong in `nativeBuildInputs`?

Reply:
You're correct, I've moved them now. I suspect this also applies to the quakespasm derivation, as this is more or less inspired of that.